src/components/views/elements/DNDTagTile.js

```diff
@@ -17,29 +17,76 @@ limitations under the License.
 
 import TagTile from './TagTile';
 
-import React from 'react';
+import React, {createRef} from 'react';
 import { Draggable } from 'react-beautiful-dnd';
+import {ContextMenu, toRightOf} from "../../structures/ContextMenu";
+import * as sdk from '../../../index';
 
-export default function DNDTagTile(props) {
-    return <div>
-        <Draggable
-            key={props.tag}
-            draggableId={props.tag}
-            index={props.index}
-            type="draggable-TagTile"
-        >
-            { (provided, snapshot) => (
-                <div>
-                    <div
-                        ref={provided.innerRef}
-                        {...provided.draggableProps}
-                        {...provided.dragHandleProps}
-                    >
-                        <TagTile {...props} />
+export default class DNDTagTile extends React.Component {
```

Comment:
this could be far simplified and kept a functional component using our `useContextMenu` hook

Reply:
Cool, taking a look